tests/failures/check/check_skip_squash.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+set -e
+
```

Comment:
Could you please state here information, that this is only temporary, and we are aiming for enabling tests for squashing for RHEL8 and RHEL9?

Reply:
RHEL8 and RHEL9 use podman instead of docker and therefore squash is deprecated.

Reply:
Deprecated? I think that podman is able to squash images. Some docs are [here](https://docs.podman.io/en/latest/markdown/podman-build.1.html#squash). Should not it be enough to rewrite the test for squashing?
Which, I understand, is not so pressing, but would be nice to have in the future.

Reply:
Thanks for posting the link. Issue is filed here: https://github.com/sclorg/container-common-scripts/issues/274, so it is not blocked.

Reply:
You are right, if podman supports squashing, then it should not be deprecated.

Reply:
OK, thanks for creating the issue.

Reply:
> OK, thanks for creating the issue. I would, though, still leave some note about it in the comment.

You mean in this pull request or where?

Reply:
See my code suggestion.